src/screens/Rooms/components/RoomAddCard.tsx

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+import {View, Text} from 'react-native';
+
+export default class RoomAddCard extends Component {
+	public render() {
+		return (
+			<View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
+				<Text>afasfasfas</Text>
```

Comment:
what is this text for?

Reply:
just for new screen, i work with it later at https://trello.com/c/ajslG1JE/31-upgrade-rooms-screen